### PR TITLE
Warn on missing value, add fallback for consistent secret param

### PR DESCRIPTION
**Pull request by mitchhentges**
I spent a few days debugging this integration not working, and it turned out to be inconstant naming: `GITHUB_APP_ID` and `GITHUB_API_SECRET`? It's like you're asking for typos :wink:

I've hardly done any python work before, so I'm not sure how well this fits into standards.
Additionally, I'm not sure if there's better warn functionality built into the Sentry integration API, so I simply log errors to `stdout`. It seems like the `OAuth2Login` actually throws the exception that is printed on the web UI, so I couldn't quite inject a more helpful error message into there.

TL;DR, I don't know what I'm doing, but I want parameter naming to be more consistent

### CODE REVIEW

File: sentry_auth_github/constants.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 CLIENT_ID = getattr(settings, 'GITHUB_APP_ID', None)
 
 CLIENT_SECRET = getattr(settings, 'GITHUB_API_SECRET', None)
+if not CLIENT_SECRET:
```
Comment (dcramer) [inline sentry_auth_github/constants.py]: This block of code shouldn't do anything. Am I missing something?

File: sentry_auth_github/provider.py [hunk 2]
```diff
@@ -21,6 +21,12 @@ class GitHubOAuth2Provider(OAuth2Provider):
     client_id = CLIENT_ID
     client_secret = CLIENT_SECRET
 
+    if not client_id:
+        print('WARNING: GITHUB_APP_ID isn\'t set')
```
Comment (dcramer) [inline sentry_auth_github/provider.py]: generally the pythonic way here would be to:

```
import warnings

...

warnings.warn('GITHUB_APP_ID is not set')
```

File: sentry_auth_github/constants.py (hunk 1 above)
Comment (mitchhentges) [inline sentry_auth_github/constants.py]: Gah, it happened again.
It should `getattr` from `GITHUB_APP_SECRET` instead. I'll fix it in a sec

File: sentry_auth_github/provider.py (hunk 2 above)
Comment (mitchhentges) [inline sentry_auth_github/provider.py]: Awesome, thanks

**Comment by mitchhentges** [comment]
Ah, sorry, didn't realize that there was a CI failure. Should be good now :)

**Comment by mitchhentges** [comment]
I'm not sure that this adds value anymore, I'll close this
